app/src/main/res/layout/custom_row_item.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
It is preferable to use a single Relative Layout instead of nested Linear Layouts